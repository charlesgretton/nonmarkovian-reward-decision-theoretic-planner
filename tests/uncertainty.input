loadWorld("uncertainty.pltl")
constraintToDot
quit
