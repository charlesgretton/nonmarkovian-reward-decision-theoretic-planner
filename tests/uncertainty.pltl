[rwd1, 1.0]? ~a

action x
    a (0.5)
endaction
