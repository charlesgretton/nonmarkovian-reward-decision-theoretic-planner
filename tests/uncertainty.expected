
digraph "Constraint specification" {
center = true;
edge [dir = none];
n0 [label="1.00" shape=box];
}


